test/datadog.spec.ts

```diff
@@ -1,537 +1,11 @@
-import { App, Stack, Token } from "aws-cdk-lib";
-import { Template } from "aws-cdk-lib/assertions";
+import { App, Stack } from "aws-cdk-lib";
```

Comment:
This file was first renamed as `datadog-lambda.spec.ts`. Then `datadog.spec.ts` was created again with one test, to ensure the `Datadog` API still works.